website/www/site/layouts/shortcodes/playground.html

```diff
@@ -10,10 +10,13 @@
 limitations under the License. See accompanying LICENSE file.
 */}}
 
-<iframe
-    src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled={{ .Get 0 }}&example={{ .Get 1 }}&code={{ .Get 2 }}"
-    width="100%"
-    height="{{ .Get 3 }}"
-    class="code-snippet playground"
-    allow="clipboard-write">
-</iframe>
+<div id="playgroundIframeContainer">
+    <iframe
+        src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled={{ .Get 0 }}&example={{ .Get 1 }}&code={{ .Get 2 }}"
```

Comment:
Better use named parameters in this shortcode since we already have 5. They will only be adding up.

Reply:
done

Reply:
Opportunity to update embedded playground source to https://play.beam.apache.org/

Reply:
done